shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -403,6 +403,10 @@ - (void)surfaceUpdated:(BOOL)appeared {
 
 #pragma mark - UIViewController lifecycle notifications
 
+- (void)viewDidLoad {
```

Comment:
This needs to call `[super viewDidLoad]` I think.

Reply:
This call is unlikely to do anything where it is, because the engine is created in viewWillAppear:, which is after this. So the channel will be nil, and the message will go nowhere.

This should be right after the launchEngine: call just below (and in the initWithEngine: for that path)